docs/configuring-a-resource.md

```diff
@@ -557,13 +559,66 @@ p.AddResourceConfigurator("aws_autoscaling_group", func(r *config.Resource) {
 })
 ```
 
+### Initializers
+
+Initializers establish ownership of the supplied Managed resources in Crossplane.
+This typically involves the operations that are run before calling any
+ExternalClient methods.
+
+This configuration option will provide that setting custom initializers for per 
+resource and is set by using the [InitializerFns] field that is a list of [NewInitializerFn]:
+
+```go
+// NewInitializerFn returns the Initializer with a client.
+type NewInitializerFn func(client client.Client) managed.Initializer
+```
+
+Initializer is an interface in [crossplane-runtime]:
+
+```go
+type Initializer interface {
+	Initialize(ctx context.Context, mg resource.Managed) error
+}
+```
+
+So, an interface must be passed to the related configuration field for adding initializers for a resource.
```

Comment:
It could be helpful to note that tagging is most probably the only use case they would be concerned as an initializer, really, so that they don't feel like it's a widely used mechanism.